mmocr/datasets/preparers/data_obtainer.py

```diff
@@ -146,10 +147,17 @@ def move(self, mapping: List[Tuple[str, str]]) -> None:
             tuple contains the source file name and the destination file name.
         """
         for src, dst in mapping:
-            src = osp.join(self.data_root, src)
-            dst = osp.join(self.data_root, dst)
-            if osp.exists(src) and not osp.exists(dst):
-                shutil.move(src, dst)
+            if '*' not in src:
+                src = osp.join(self.data_root, src)
+                dst = osp.join(self.data_root, dst)
+                if osp.exists(src) and not osp.exists(dst):
+                    shutil.move(src, dst)
+            else:
+                # use fuzzy search to find all the files
+                src_files = glob.glob(osp.join(self.data_root, src))
+                for src_file in src_files:
+                    dst_file = osp.join(self.data_root, dst)
+                    shutil.move(src_file, dst_file)
```

Comment:
#1639 happens to implement a similar but more concise version in:
https://github.com/open-mmlab/mmocr/blob/bda6569cb730c722bdef12ed2a62d6c54f1cfd23/mmocr/datasets/preparers/data_obtainer.py#L150-L160

Would you simply adapt these lines to your PR to avoid future merge?